starport/cmd/network_chain_launch.go

```diff
@@ -0,0 +1,67 @@
+package starportcmd
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagRemainingTime = "remaining-time"
+)
+
+// NewNetworkChainLaunch creates a new chain launch command to launch
+// the network as a coordinator.
+func NewNetworkChainLaunch() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "launch [launch-id]",
+		Short: "Launch a network as a coordinator",
+		Args:  cobra.ExactArgs(1),
+		RunE:  networkChainLaunchHandler,
+	}
+
+	c.Flags().String(flagRemainingTime, "", "The remaining time to launch the chain")
+	c.Flags().AddFlagSet(flagNetworkFrom())
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+
+	return c
+}
+
+func networkChainLaunchHandler(cmd *cobra.Command, args []string) error {
+	nb, err := newNetworkBuilder(cmd)
+	if err != nil {
+		return err
+	}
+	defer nb.Cleanup()
+
+	// parse launch ID.
+	launchID, err := strconv.ParseUint(args[0], 10, 64)
+	if err != nil {
+		return errors.Wrap(err, "error parsing launchID")
+	}
+	if launchID == 0 {
+		return errors.New("launch ID must be greater than 0")
+	}
+
+	remainingTime, _ := cmd.Flags().GetUint64(flagRemainingTime)
```

Comment:
We use the value that represents the number of seconds on `spn` to easily manage the Unix timestamp but it may not be user friendly since the remaining time will generally be a few days. Maybe we can think of a format to pass number of days and hours.

Also the `minLaunchTime` and `maxLaunchTime` params of the `launch` module represent the range for the time to set in second. If the command fails we could fetch them through the `Params` query and show the range in a user friendly format with number of days, hours, etc..

We don't need to implement these for this PR but we should create an issue for it I think

Reply:
Great, I created a package for we can print the timestamps, and also I changed the remaining time parameter to the duration for we can use it more easily 
e.g.:
```
starport n --local chain launch 1 --remaining-time 4h
starport n --local chain launch 1 --remaining-time 10d
```